.github/ISSUE_TEMPLATE/integration_test_request.md

```diff
@@ -0,0 +1,43 @@
+If you are interested in access to OpenLab resources, please fill out the details below. You can always find more information about OpenLab at https://openlabtesting.org

```

Comment:
Entire file needs to change to the following; simple copy and paste is sufficient:

If you are interested in access to OpenLab resources, please fill out the details below. You can always find more information about OpenLab at https://openlabtesting.org

If you are just filing an issue, ignore/delete these fields and file your issue.

### Full Name

### Email

### Company/Organization

### Job Title

### Phone

### What is your focus?
- [ ] Performance
- [ ] Scale
- [ ] Reliability
- [ ] Resiliency
- [ ] Integration
- [ ] Multi-Cloud
- [ ] Operational Tooling
- [ ] Edge or IoT
- [ ] Machine Learning
- [ ] Artificial Intelligence
- [ ] Application Ecosystem
- [ ] Other


### If this is for an open source project what is it?

### Brief project description

### Is project code 100% open source? If so, what is the URL or URLs where it is located?

### What kind of machines (VMs or Baremetal) and how many do you expect to use?

### What OS are you planning to use?

### Any special network configuration you expect or anticipate implementing?

### Any architecture or other specifications/requirements (CPU, RAM, GPU, etc)?

### Please state your contributions to the open source community and any other relevant initiatives.

### What testing are you planning to implement or need assistance implementing?

### How will this testing advance application and/or tooling built on-top of open infrastructure?

### Will you publish blog or paper from your testing?

- [ ] Yes
- [ ] No - if no, how will you present your results?

### Any other relevant details we should know about while preparing the infrastructure?

Reply:
1. Yes, reporting bug and submitting test request maybe come from different actual role, so split to two templates to make new member focus on thing they care about.
2. We should add Github account id into joining template in order to we can use it to invite new member into OpenLab github organization.
3. Please mark `job title` and `phone` as optional, I think not everybody would like to show this personal information.